docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -77,6 +77,9 @@ like `tctl edit`. With touch required, hardware key support provides better secu
   so we recommend only using one PIV application at a time.
 </Admonition>
 
+- Running Smart Card daemon:
```

Comment:
This is the only mention of "Smart Card daemon" on this page, so I would add a sentence about how hardware key support uses the smart card daemon. (Is "Smart Card daemon" the product name, or would it make sense to lowercase it?)

Reply:
Changed to "smart card driver" which is more universal